### DISCUSSION TITLE
global keyboard shortcuts to apply a profile

### DISCUSSION THREAD

**Discussion by the-dbp** [snappysnappydog/Simple-IP-Config#142] [Ideas]
I have to constantly switch between my auto profile and a static ip profile, it would be awsome if I could just hit a keyboard shortcut to do this without opening simple ip config

**Comment by KurtisLiggett** [comment]
Maybe a tray menu item to set DHCP?
I do have setting profiles from the tray on the backburner, but maybe this would be a good first step.
